dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -358,3 +251,119 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         }
     }
 }
+
+namespace winrt::Microsoft::Windows::PushNotifications::factory_implementation
+{
+
+    inline constexpr auto c_maxBackoff{ 5min };
+    inline constexpr auto c_initialBackoff{ 60s };
+    inline constexpr auto c_backoffIncrement{ 60s };
+
+    const HRESULT WNP_E_NOT_CONNECTED = static_cast<HRESULT>(0x880403E8L);
+    const HRESULT WNP_E_RECONNECTING = static_cast<HRESULT>(0x880403E9L);
+    const HRESULT WNP_E_BIND_USER_BUSY = static_cast<HRESULT>(0x880403FEL);
+
+    bool PushNotificationManager::IsChannelRequestRetryable(const hresult& hr)
```

Comment:
Thinking about this a bit more, there is no reason that IsChannelRequestRetryable should be a member of PushNotificationManager. 
Recommend: Remove this API from the header file and just call it IsChannelRequestRetryable here so that it is a global function for this namespace